textile/src/main/java/io/textile/textile/LifecycleService.java

```diff
@@ -66,7 +66,11 @@ void stopNodeImmediately() {
             timer = null;
         }
         Textile.instance().stop();
-        nodeStoppedListener.onNodeStopped();
+        // Sometimes this function is run while nodeStoppedListener is still null, crashing the app
+        if (timer != null) {
```

Comment:
Why check if `timer` is `null` instead of `nodeStoppedListener`?

Reply:
+ 1 for the question

Reply:
Doh, copy pasta. Will fix